stdlib/Sockets/src/Sockets.jl

```diff
@@ -539,7 +539,19 @@ end
 """
     connect([host], port::Integer) -> TCPSocket
 
-Connect to the host `host` on port `port`.
+Connect to the host `host` on port `port`. The default `host` is `localhost`.
+
+# Examples
+```jldoctest
+julia> server = listen(9000);
+
+julia> socket = connect(9000)
+TCPSocket(RawFD(25) open, 0 bytes waiting)
+
+julia> close(socket)
```

Comment:
We should provide a brief explanation of why we would want to `close` these objects in general (not necessarily just for the purpose of the doctests).